### ISSUE TITLE
Bad monikers aren't normalized (even though they should be)

### ISSUE THREAD

**Issue by gabebw** [gabebw/croniker#83]
To reproduce:

* Enter an invalid moniker (like `<<<<<`) and submit the form
* There is a correct error message: "Monikers cannot be blank"
* However, the moniker is still displayed as `<<<<<` rather than a blank string

The `fmap normalizeMaybe` in `Form/Profile.hs` should fix this, but it's evidently not.

**Comment by gabebw** [comment]
After #86, this doesn't make as much sense anymore. Closing in favor of the more targeted #87.
